src/libraries/System.Security.Cryptography.X509Certificates/src/ExculdeApiList.txt

```diff
@@ -0,0 +1,3 @@
+// exclusions to work around browser builds
```

Comment:
This is not for browser.  It's for now for RID-less.  Better thing to comment here is why you need to exclude this along with a link to the bug tracking a fix to the tooling so that this can be removed.

Reply:
Actually, given this is only for getting nullable correct on the generated interface member it might be easier to just nowarn the compiler message around inconsistent nullable annotations.  The annotations here are irrelevant since its runtime assembly (and an explicit interface).  /cc @safern 

Still add a comment about why the nowarn with a pointer to the genAPI issue.